yapapi/executor/__init__.py

```diff
@@ -451,26 +468,33 @@ async def accept_payment_for_agreement(
             events.PaymentAccepted(agr_id=agreement_id, inv_id=inv.invoice_id, amount=inv.amount)
         )
 
-    def approve_agreement_payments(self, agreement_id):
+    def approve_debit_notes_for_agreement(self, agreement_id):
```

Comment:
Perhaps this should be renamed to `accept_debit_notes_for_agreement` for the sake of consistency with `accept_payments_for_agreement`? (also the type hint for `agreement_id` is missing)

Reply:
fixed in 761f820